src/Features/Core/Portable/EditAndContinue/EditAndContinueMethodDebugInfoReader.cs

```diff
@@ -24,6 +24,9 @@ namespace Microsoft.CodeAnalysis.EditAndContinue
     /// </summary>
     internal abstract class EditAndContinueMethodDebugInfoReader
     {
+        // TODO: Remove, the path should match exactly. Workaround for https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1830914.
+        internal static bool IgnoreCaseWhenComparingDocumentNames;
```

Comment:
seens pretty hacky?  maybe better to just wait on the fix coming down the line?

Reply:
Of course it's hacky. I don't think we should block the entire feature on this.

Reply:
my point was that this might not be necessary if, say, the PS fix was a day away :)   If it's more like several weeks out, then a hacky mitigation might be appropriate :)

Reply:
It is not a day away AFAICT